src/test/kotlin/org/rust/ide/typing/RustRawLiteralHashesDeleterTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.typing
+
+class RustRawLiteralHashesDeleterTest : RustTypingTestCaseBase() {
+    override val dataPath = "org/rust/ide/typing/rawHashes/fixtures"
+
+    // Swap file names, so we can reuse tests for inserter :-D
+    override fun beforeFileName(fileName: String): String {
+        return super.afterFileName(fileName)
+    }
+
+    override fun afterFileName(fileName: String): String {
+        return super.beforeFileName(fileName)
+    }
```

Comment:
I decided to stick with `val inverse: Boolean` solution, as it looks much clearer for me. InlineFile is not fine imho as `/*|*/` solution still interrupts lexer.